net-libs/nodejs/nodejs-22.4.1.ebuild

```diff
@@ -94,8 +94,6 @@ src_prepare() {
 	# Avoid writing a depfile, not useful
 	sed -i -e "/DEPFLAGS =/d" tools/gyp/pylib/gyp/generator/make.py || die
 
-	sed -i -e "/'-O3'/d" common.gypi node.gypi || die
```

Comment:
Why is this being deleted? I find it hard to believe -O3 is safer than -O2, and if anything, that's a total coincidence and could easily change between releases and may be dependent on the order of passes.

https://issues.chromium.org/issues/42204492#comment11 also implies that it hasn't really been diagnosed correctly.

This really needs some proper investigation. I'm OK with a workaround but this isn't the right one.

_If_ we were to do what you're doing, we'd need to append-flags -O3 or something instead, not just filter -O2 (which isn't going to work correctly), but that's not a good approach.

Please try to identify which option causes it by diffing -O2 and -O3.